test/matrix.cc

```diff
@@ -256,7 +256,7 @@ TEST(Matrix, test_matrix) {
       ++found;
     }
   }
-  EXPECT_EQ(found, 10) << " not the number of results as expected";
+  EXPECT_EQ(found, 11) << " not the number of results as expected";
```

Comment:
this tested that we find less connections once we decrease the max matrix distance in the config. seems there was a close case and a* actually finds it because it's better directed.